app.js

```diff
@@ -110,27 +114,67 @@ function getHashCode(str) {
   return hash;
 }
 
+function dedent(str) {
+  // Reduce the indentation of all lines by the indentation of the first line
+  const match = str.match(/^\n?( +)/);
+  if (!match) {
+    return str;
+  }
+  const indentRe = new RegExp(`\n${match[1]}`, 'g');
+  return str.replace(indentRe, '\n').replace(/^\n/, '');
+}
+
+function formatOptionValue(key, value) {
+  if (typeof value === 'string') {
+    return value;
+  }
+  if (CERT_OPTIONS.includes(key)) {
+    return chalk`${
+      value.toString()
+        .replace(/-----.+?-----|\n/g, '')
+        .substring(0, 80)
```

Comment:
Before this change key / cert was printed as something like `<Buffer 2d 2d 2d 2d 2d 42 ... >`

Reply:
nice addition